app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTestWithAuth.kt

```diff
@@ -0,0 +1,102 @@
+package org.simple.clinic.patient
+
+import android.arch.core.executor.testing.InstantTaskExecutorRule
+import android.support.test.runner.AndroidJUnit4
+import com.google.common.truth.Truth.assertThat
+import io.reactivex.Observable
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.simple.clinic.AppDatabase
+import org.simple.clinic.AuthenticationRule
+import org.simple.clinic.TestClinicApp
+import org.simple.clinic.TestData
+import org.simple.clinic.bp.BloodPressureRepository
+import org.simple.clinic.facility.FacilityRepository
+import org.simple.clinic.user.UserSession
+import javax.inject.Inject
+
+@RunWith(AndroidJUnit4::class)
+class PatientRepositoryAndroidTestWithAuth {
```

Comment:
This test does not seem to require login to the QA server. I feel we can put this in the `PatientRepositoryAndroidTest` and just create a local user who is not `APPROVED_FOR_SYNCING`.

Reply:
Done